src/k-provider/ovp/external-captions-builder.js

```diff
@@ -13,14 +13,15 @@ const CaptionsFormatsMap: {[format: string]: string} = {
 };
 
 class ExternalCaptionsBuilder {
-  static createConfig(captions: Array<Object>): Array<PKExternalCaptionObject> {
+  static createConfig(captions: Array<Object>, ks: string): Array<PKExternalCaptionObject> {
     return captions.map(caption => {
       let url = caption.url;
       let type = CaptionsFormatsMap[caption.format];
       if ([KalturaCaptionType.DFXP, KalturaCaptionType.CAP].includes(caption.format)) {
         url = caption.webVttUrl;
         type = CaptionsFormatsMap[KalturaCaptionType.WEBVTT];
       }
+      url = ks ? url + '/ks/' + ks : url;
```

Comment:
please add test

Reply:
need to verify if url already has query string params, e.g. contains `?` and if so add it as `&ks=KS`

Reply:
@yairans tests added

Reply:
@OrenMe it worked also as ?ks=ks and /ks/ks